addon/components/one-way-input.js

```diff
@@ -77,7 +77,11 @@ const OneWayInputComponent = Component.extend({
     if (this._sanitizedValue !== value) {
       this._sanitizedValue = value;
       if (action) {
-        action(value);
+        if (typeof action === 'string') {
```

Comment:
That add-on already handles string actions https://github.com/martndemus/ember-invoke-action/blob/master/addon/index.js#L14, could you also implement like it's already done in https://github.com/DockYard/ember-one-way-controls/blob/master/addon/components/one-way-select.js#L104